### PR TITLE
Create calculator component

### PR DESCRIPTION
**Pull request by Draxeytina**
image

- Add and update calculator.js
- Add and update calculator.css
- remove unnecessary functions
- Update the main js file to render calculator app

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import './index.css';
-import App from './App';
+import Calculator from './components/calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (MahmoudBakr23) [inline src/index.js]: - [x] Please make sure to make the wrapper of the Calculator component is the App component. 🙏 Also, make sure that the index.js file is only to wrap the Parent Component 'App.js'

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Draxeytina  😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Nice design
- Correct Linters
- Great job managing to set up the calculator component by importing the two logic files and handling the buttons
However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] Please make sure to remove the boilerplate code from the `App.js` file and turn it into a class-based component as it's a requirement in this project.

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Draxeytina) [reply src/index.js]: Right, I had taken a shortcut 😆 Thanks @MahmoudBakr23

**Review by rotimiazeez — APPROVED** [review]
# Hi @Draxeytina 🙋🏽‍♂️, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Draxeytina** [comment]
> # Hi @Draxeytina 🙋🏽‍♂️,
> ### Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you @rotimiazeez !

**Pull request merged**